src/OrbitBase/ThreadUtilsLinux.cpp

```diff
@@ -51,4 +51,10 @@ void SetCurrentThreadName(const std::string& thread_name) {
   }
 }
 
+pid_t GetCurrentProccessId() {
```

Comment:
you have extra 'c' in Process.

Reply:
I should be more careful!!! Thanks so much for pointing out this.